### ISSUE TITLE
Limit LN fee to what on-chain fee would be

### ISSUE THREAD

**Issue by Giszmo** [ACINQ/phoenix#576]
I'm using Phoenix as my main driver and I love it but I find myself doing on-chain transactions more than LN recently as on-chain fees are low and I send substantial amounts.

For example 0.5% of $800 is clearly more expensive than 7sat/vB today.

I would love to see a fee guarantee to be lower than the on-chain fees. It would get more fees your way and would make using Phoenix even easier.

Of course you can probably do some chain analysis to see how many people actually avoid paying the LN fees in this way and for which amount of payment but I think it's the right thing to do regardless of how often people opt for on-chain.

**Comment by tohrxyz** [comment]
Free market and i love what product acinq provides, clearly the best out there, but 0.4% is simply too much.

+1 for lowering fees

**Comment by dpad85** [comment]
Choosing to pay with Lightning or on-chain is not just about the fees. Lightning is not just a cheaper on-chain, it brings valuable features: a payment made over Lightning is instant and final. The payment is also more private as it is not publicly broadcast and visible by everyone.

Lightning can also be cheaper down the line : an on-chain transaction may have to be bumped for higher fees (the fee estimation does not guarantee a timely confirmation), and will also have to be consolidated at some point at an unknown fee rate (whereas on Lightning you always have 1 single utxo).

That said, we plan to display what fee would be applied on the *other* method. That is, if you are paying with Lightning, the UI would indicate that the fee would be X on-chain.

**Comment by Giszmo** [comment]
@dpad85 I agree there is more than just fees in favor of LN but for ACINQ there is one fee that you can charge and one  you can't charge so I would appreciate if you could adapt to the most fee-sensitive users so that all of us can comfortably opt for LN always without having to price our privacy and confirmation speed. Having to think about there maybe being a better way of paying with this same wallet is just a bad user experience.

And while at the levels I'm paying regularly, 0.4% is expensive, too, this issue is not about the 0.4% as a basic fee. It's just about not stressing the user with the possibility to save on fees by using an inferior tech.
